datasource_api_client/models/list_request.py

```diff
@@ -58,6 +64,8 @@ def to_dict(self) -> Dict[str, Any]:
 
     @classmethod
     def from_dict(cls: Type[T], src_dict: Dict[str, Any]) -> T:
+        from ..models.datasource_config import DatasourceConfig
```

Comment:
I know this is auto-generated but just curious, why would they add the import inside the method?

Reply:
This is under the class method. And the data type is only used for converting from dictionary. I think the rational here is for lazy imports.